pipelines/broad/rna_seq/RNAWithUMIsPipeline.wdl

```diff
@@ -30,12 +30,12 @@ workflow RNAWithUMIsPipeline {
     String read2Structure
     String output_basename
 
-    # The following inputs are only required if fastqs are given as input.
-    String? platform
-    String? library_name
-    String? platform_unit
-    String? read_group_name
-    String? sequencing_center = "BI"
+    # sato: make a note of this change---no longer optional
+    String platform
```

Comment:
For the 'parameter meta' for all these values we can remove the 'only required when using fastq files as input blurb'

Reply:
Yes